troubleshooting/incrementally_configuring_images.md

```diff
@@ -0,0 +1,297 @@
+# Incrementally configuring images
```

Comment:
This should be capitalized (the main page header is the only header where the style guide indicates to capitalize throughout -- the rest use sentence casing, like you have for this one)

```suggestion
# Incrementally Configuring Images
```